README.md

```diff
@@ -90,3 +91,13 @@ things simple. The library comes in 4 parts:
    started with a provider in go.[^1]
 
 [^1]: The "Hello, Pulumi" example shows the `infer` layer.
+
+
+## Generating SDKs and schema
+
+In order to use the provider in Pulumi programs, you need to generate at least one SDK.
```

Comment:
There is a reason we encourage testing in Pulumi YAML: no SDKs need be generated.